tools/rsictl/src/subcmds.rs

```diff
@@ -1,6 +1,10 @@
 use crate::tools::{self, hexdump};
 use clap::{Args, ValueEnum};
 
+const METADATA_SIGNATURE_SIZE: usize = 96;
+const METADATA_SIZE: usize = 0x150;
+const SIGNED_METADATA_SIZE: usize = METADATA_SIZE + METADATA_SIGNATURE_SIZE;
```

Comment:
I think that this constants should be put in rust-rsi crate. Rsictl is just a comandline tool utilizing rust-rsi API. Why not make these constants available for all in the lib crate?